java/java.completion/test/unit/data/org/netbeans/modules/java/completion/data/SimpleLambdaExpression2Start.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package test;
+
+public class Test {
+@interface TestAnnotation {
+}
+    interface Foo {
+        int op (@TestAnnotation String s, int i, int j);
+    }
+
+    private void test(Foo f) {
+    }
+    
+    private void test2(String str, int num, Foo f) {
+    }
+    
+    public static void main(String[] args) {
+	Test t = new Test();
```

Comment:
Closing bracket missing?

Reply:
It was deliberately done as I want to add code inside main method. If I add closing bracket then test code will be added after the main method which I don't want